src/components/home/Join.jsx

```diff
@@ -1,5 +1,14 @@
+import Title from "../Title";
+import Social from "./Social";
 const Join = () => {
-  return <div>Join</div>;
+  return (
+    <div className="p-10">
+      <Title subTitle="JOIN US AND GET INVOLVED" title="JOIN US" />
+      <div className="p-10">
```

Comment:
remove this div and you can apply "flex flex-col gap-[some number]" in the most parent one if you need space between these two components.